kqml/kqml_module.py

```diff
@@ -8,30 +8,32 @@
 from docutils.io import InputError
 
 def translate_argv(raw_args):
-    '''Enables conversion from system arguments.
-    
-    Inputs:
-    ------
-    raw_args -- args taken raw from the system input.
-    
-    Outputs:
+    """Enables conversion from system arguments.
```

Comment:
We can follow the docstring conventions in INDRA wherever new documentation is added.